oak_functions/sdk/oak_functions/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "oak_functions"
+version = "0.1.0"
+authors = ["Razieh Behjati <razieh@google.com>"]
+edition = "2018"
+license = "Apache-2.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[dependencies]
+oak_functions_abi = "0.1.0"
```

Comment:
Please use relative paths when referring to local crates. I know there are overrides somewhere in the manifest files, but those tend to break easily, especially if someone at some point registers an actual crate with this name on crates.io.

Reply:
I think I've been specifying it explicitly, since I got into issues otherwise. We can do another pass at some point, but definitely this is not the only place in which it would be done.

Reply:
Done.